drivers/gree_cooper_hunter_hvac/driver.js

```diff
@@ -35,6 +48,7 @@ class GreeHVACDriver extends Homey.Driver {
             data: {
                 id: message.cid,
                 mac: message.mac,
+                ipAddress: remoteInfo.address,
```

Comment:
Because auto-discovery does not work for every device an IP address is added to the device's data which the `Finder` service will also scan for devices.